projects/core/src/cart/facade/cart-util.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Cart } from '../../model/cart.model';
+
+export class CartUtilService {
+  constructor() {}
+
+  static isCreated(cart: Cart): boolean {
+    return (
+      cart &&
+      !!Object.keys(cart).length &&
+      (typeof cart.guid !== 'undefined' || typeof cart.code !== 'undefined')
```

Comment:
why we need check this `(typeof cart.guid !== 'undefined' || typeof cart.code !== 'undefined')`? It is impossible that there is cart without code or guid.

Reply:
Cart guid and core are initialized now by the local storage. But as discussed offline, guid always is a sign of created cart. So check for cart.code may be dropped.